server/model/monitor.js

```diff
@@ -96,11 +96,15 @@ class Monitor extends BeanModel {
             screenshot = "/screenshots/" + jwt.sign(this.id, UptimeKumaServer.getInstance().jwtSecret) + ".png";
         }
 
+        const path = await this.getPath();
+        const pathName = path.join(" / ");
+
         let data = {
             id: this.id,
             name: this.name,
             description: this.description,
-            pathName: await this.getPathName(),
+            path,
+            pathName,
```

Comment:
I wanted to avoid adding logic into the `let data = { ... }`.
For all other fields, we either fetch the data via an async method or we assign a class-field. Fetching `path` and creating `pathName` above (hopefully) keeps things cleaner.